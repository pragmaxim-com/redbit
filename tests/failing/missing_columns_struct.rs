
#[entity]
struct MissingColumnsStruct {
    #[pk]
    id: ParentPK,
}
