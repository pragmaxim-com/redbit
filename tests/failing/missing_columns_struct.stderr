error: cannot find attribute `entity` in this scope
 --> tests/failing/missing_columns_struct.rs:2:3
  |
2 | #[entity]
  |   ^^^^^^
  |
help: consider importing this attribute macro
  |
2 + use redbit::entity;
  |

error: cannot find attribute `pk` in this scope
 --> tests/failing/missing_columns_struct.rs:4:7
  |
4 |     #[pk]
  |       ^^

error[E0412]: cannot find type `ParentPK` in this scope
 --> tests/failing/missing_columns_struct.rs:5:9
  |
5 |     id: ParentPK,
  |         ^^^^^^^^ not found in this scope

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/failing/missing_columns_struct.rs:6:2
  |
6 | }
  |  ^ consider adding a `main` function to `$DIR/tests/failing/missing_columns_struct.rs`
